Gemfile.lock

```diff
@@ -760,9 +760,3 @@ DEPENDENCIES
   whenever
   wicked_pdf
   wkhtmltopdf-binary
-
-RUBY VERSION
-   ruby 2.1.5p273
-
-BUNDLED WITH
-   1.15.1
```

Comment:
Wonderful! Thank you for this information! I'm using Bundler version 1.7.6 and gem version 2.4.3,  though I am using ruby 2.1.5. New to Ruby world, so I hope that helps!